supertokens_python/recipe/thirdpartyemailpassword/utils.py

```diff
@@ -33,14 +39,16 @@
 
 
 class InputEmailVerificationConfig:
+    # TODO: FIXME
+    # FIXME: Why is this duplicate of emailpassword.utils.InputEmailVerificationConfig?
```

Comment:
TODO

Reply:
What should I do for this one? Just import and re-export `InputEmailVerificationConfig`?

Reply:
I dont think you can import and export from emailpassword recipe since the type of user in `create_and_send_custom_email` function would be different.

Reply:
Removed the comment.